dev/ci/ci-basic-overlay.sh

```diff
@@ -127,3 +127,9 @@
 ########################################################################
 : ${tlc_CI_BRANCH:=master}
 : ${tlc_CI_GITURL:=https://gforge.inria.fr/git/tlc/tlc.git}
+
+########################################################################
+# Bignums
+########################################################################
+: ${Bignums_CI_BRANCH:=master}
+: ${Bignums_CI_GITURL:=https://github.com/coq/bignums.git}
```

Comment:
minor nit, it should be `bignums_CI_BRANCH` etc... as I hope soon much of this boilerplate is handled automagically, but the name should correspond to the entry in `ci-bignums`